SCI-FI_FAIR/presenter-lazercatbot.md

```diff
@@ -1,17 +1,37 @@
 # What is the NAME of your project?
-P2P Lazer Catbot
+P2P Lazer Catbot, an interplanetary network for Cats
 
 # What DOES THE PROJECT DO/what will you SHOW?
-High-tech kitty entertainment, made p2p-capable at the Elective Session earlier
+
+This project aim to bridge lazer, Cats and the internet.
+
+From the early days to its current form it used bleeding edge technology, so it now need to be distributed, this session will focus on the physical part of the project.
+
+It will feature amazing activities like:
+
+- use IKEA grade build instruction
+- understand how your console D-pad works
+- learn how to connect your kettle to the internet
+
+Some may say this fair session is an excuse to have more cats entertained, this is the truth.
+
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/6NYyGC-wZKU" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
This iframe is not rendering on Github :(

Reply:
removed !